docker/README.md

```diff
@@ -0,0 +1,128 @@
+# Dockerizing
+
```

Comment:
## When to use Docker

Any developer is welcome to use Docker in their development setup. New apps developed at Abtion should not require to have a docker setup unless there is a good reason for it (like an uncommon dependency).
Our own legacy apps might require a docker setup, however, we will strive to not make it a requirement if possible.